crates/router/src/core/payments/helpers.rs

```diff
@@ -1243,15 +1243,16 @@ pub fn create_authorize_url(
 pub fn create_webhook_url(
     router_base_url: &String,
     merchant_id: &id_type::MerchantId,
-    connector_name: impl std::fmt::Display,
+    merchant_connector_id_or_connector_name: &String,
```

Comment:
```suggestion
    merchant_connector_id_or_connector_name: &str,
```

Then we can avoid the `to_string()` calls.